src/Microsoft.AspNetCore.Mvc.Core/Filters/MiddlewareFilterAttribute.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.AspNetCore.Mvc.Internal;
+using Microsoft.Extensions.DependencyInjection;
+using System.Reflection;
+
+namespace Microsoft.AspNetCore.Mvc.Core.Filters
+{
+    [AttributeUsage(AttributeTargets.Class | AttributeTargets.Method, AllowMultiple = false, Inherited = true)]
```

Comment:
Were you able to allow multiple? One on the Class, one on the Action, and have them run one after the other?